Gives Warrant Officer to: BC , Biomech Tech

Guidesu opened the pull request:
Second PR
I talked with Alanii in DMs
They told me 3 jobs to put Warrant Officers in.

Brig Chief:
NTEF and SMC:
W1           W1
W2           W2
W3           W3

Biomechanical Technician:
NTEF and SMC:
W2           W2
W3           W3


🆑 ADD: Gives specialized Roles WO as a rank

Guidesu committed:
First PR: Gives Chaplain more Fluffy Ranks.

Hello i am Guidesu this is my FIRST PR
Please if you can teach me more how to code or fix bugs i would be glad.
Thank you
I added a few ranks to Chaplain:
E6
E7
W1
W2
In the NTEF and SMC

Guidesu committed:
Revert "First PR: Gives Chaplain more Fluffy Ranks."

This reverts commit aa6df0d78c2e78b44c0a0a97a109fd7289f94fd6.

Guidesu committed:
Update Chaplain ranks

Guidesu committed:
Alanii Request

Guidesu committed:
Revert "Update Chaplain ranks"

This reverts commit 1b522642f078fdab599499cb48bf6d42733b16fa.

io-util commented on maps/torch/job/medical_jobs.dm:
```diff
@@ -61,11 +61,15 @@
 		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/medical/biomech/marine,
 		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/medical/biomech/fleet,
-		/datum/mil_branch/civilian = /decl/hierarchy/outfit/job/torch/crew/medical/contractor/biomech
+		/datum/mil_branch/civilian = /decl/hierarchy/outfit/job/torch/crew/medical/contractor/biomech,
```
The `list` ends here, there does not have to be a comma at the end.

io-util replied:
Same here, comma is still there.

io-util commented on maps/torch/job/torch_jobs_boh.dm:
```diff
@@ -298,12 +300,18 @@
 		/datum/mil_rank/marine_corps/e6,
 		/datum/mil_rank/marine_corps/e7,
-		/datum/mil_rank/marine_corps/e8
+		/datum/mil_rank/marine_corps/e8,
```
Same here, `list` ends so there actually does not need to be a comma here.

Guidesu replied:
Okay thank  you

io-util replied:
The comma is still there.

gilvian commented:
No. Scientists are not supposed to be WOs ever. Just in a sake of "oh we need more ranks to RP" PR since your chaplain one made a bit of noise dosent mean every single role should get new ranks balances.

io-util reviewed (changes requested):
Code is totally fine, I am being really "strict" with this, just remove those redundant commas at the end.

Guidesu commented:
> No. Scientists are not supposed to be WOs ever. Just in a sake of "oh we need more ranks to RP" PR since your chaplain one made a bit of noise dosent mean every single role should get new ranks balances.

That fine.
I am kinda uhm.. new learning and stuffs.
I just wanted more roles to have WO...
Its not like every single role get new ranks

Guidesu committed:
Update torch_jobs_boh.dm

Guidesu commented:
Anyway thank you for your feedback pal

Guidesu commented:
There i fixed it.
I.. please just dont be rude..

io-util commented on maps/torch/job/torch_jobs_boh.dm:
```diff
@@ -265,7 +265,7 @@
 	)
 	allowed_ranks = list(
-		/datum/mil_rank/fleet/o1,
+        /datum/mil_rank/fleet/o1,
```
This line got no proper indentation anymore, you kinda messed that up a bit, just remove the additional spaces.

Guidesu replied:
Loorey.
I didint get it
I didint put the additional spaces and I cant find it
![image](https://user-images.githubusercontent.com/67030229/99881466-c613c400-2bf8-11eb-9457-1fddd318aedb.png)

io-util replied:
![image](https://user-images.githubusercontent.com/39163886/99881526-aee5ce00-2c1a-11eb-91b6-dc04ce80b0b4.png)
Line 268 in `maps/torch/job/torch_jobs_boh.dm`, check if it's really tabs and not spaces, Code Checks are failing because of it too.

Guidesu replied:
ooooh i used space
I used tab now

Guidesu replied:
its fixed thank you

io-util replied:
![image](https://user-images.githubusercontent.com/39163886/99881674-5ebb3b80-2c1b-11eb-9692-a3bde333df80.png)
It's still missing a second tab.

Guidesu replied:
okay

Guidesu replied:
Sorry its to many details and i still learning

Guidesu replied:
But thank you anyway

Guidesu committed:
Fixing the Comma like Loorey said

Thank you man

Guidesu commented:
Also thank you for helping me!
I fixed the Commas like you said
But the spare spaces on that line... i removed it but it don't update on  my github

Guidesu committed:
Fixing the spaces error

Guidesu committed:
Putting two tabs now

Guidesu commented:
Anything else i need fix?

io-util reviewed (approved):
Looks fine code wise.
